app/presenters/licences/view-licence-returns.presenter.js

```diff
@@ -96,8 +98,22 @@ function _status(returnLog) {
   // 'today' would be flagged as overdue when it is still due (just!)
   today.setHours(0, 0, 0, 0)
 
-  if (status === 'due' && dueDate < today) {
-    return 'overdue'
+  if (status === 'due') {
```

Comment:
We could invert this condition to make it a guard clause so we don't need to have nested ifs, ie:

```js
if (status !== 'due') {
  return status
}
```

Reply:
I want to sometimes return the status when it's due as well so this change doesn't remove the need to still return status at the end